project-has-many-members

From: Bryan Larsen <bryan@larsen.st>

Let's do the other ends of those two belongs-to associations. In the Project model:

SHOW_PATCH
---
 app/models/project.rb |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/app/models/project.rb b/app/models/project.rb
index 5bce2d9..d828958 100644
--- a/app/models/project.rb
+++ b/app/models/project.rb
@@ -14,6 +14,10 @@ class Project < ActiveRecord::Base
   children :stories
 
   belongs_to :owner, :class_name => "User", :creator => true, :inverse_of => :projects
+  has_many :memberships, :class_name => "ProjectMembership", :dependent => :destroy, :inverse_of => :project
+  has_many :members, :through => :memberships, :source => :user
+
+  belongs_to :owner, :class_name => "User", :creator => true, :inverse_of => :projects
 
   # --- Permissions --- #
 
